### DISCUSSION TITLE
How do we make a particular contract specific mock while working on our projects!

### DISCUSSION THREAD

**Discussion by Shourya4641** [Cyfrin/foundry-full-course-cu#2824] [General]
As part of foundry-fund-me, we have a mock smart contract for `AggregatorV3Interface` that we can use for testing in our local environment. But in the future, we will be working on different projects for which we won't have already prepared mocks. In that situation, what do we do, or how do we deal with this?!

**Comment by EngrPips** [comment]
You will have to prepare your own mock that suite your situation I assume.
